README.md

```diff
@@ -1,4 +1,86 @@
-Chitter Challenge
+# RACHEL GRIFFITHS
+------------
+## CHALLENGE: CHITTER
+------------
+* Join the new social media craze, Chitter!
+* Post a peep!
+* See all the peeps and the time they were posted.
+* Sign up to Chitter.
+* If you're lucky, you can log in and out of Chitter.
+* If you're really really lucky, you will receive an email if you are tagged in a peep.
+
+### GETTING STARTED
+------------
+* Ruby 2.5.0
+* Git Clone: git@github.com:rachjgriff/chitter-challenge.git
+* Install dependences: bundle
+
+### USAGE
+------------
+* Start command: Rackup
+* URL: http://localhost:9292/peeps
+
+### TESTING
+------------
+* rspec (via capybara)
+* 99.56% test coverage achieved
+* Linter: rubocop (1 offence)
+
+### DATABASE
+------------
+* Connect to psql
+* Follow the below instructions to create the required database.
+
+```
+CREATE DATABASE "Chitter";
+\c Chitter;
+```
+```
+CREATE DATABASE "Chitter_Test"
+\c Chitter_Test;
+```
+
+* Run the following queries in db/migrations to create the required tables in each database:
+  * 01_create_peeps_table.sql
+  * 02_create_members_table.sql
+
+### WEBSITE SPECIFICATION (REQUIREMENTS)
+------------
+* Post a message (Peep) to Chitter
+* See all Peeps in reverse chronological order with username and name displayed
+* See the time at which a Peep was posted
+* Sign up for Chitter with your name, email, username and password (username and email are unique)
+* Log in to Chitter
+* Log out of Chitter
+* Tag a user in a Peep
+* Receive an email if you are tagged in a Peep
+
+### APPROACH
+------------
+* Diagramed MVC Model of Chitter
+* Diagramed Class Responsibility Collaborator (CRC) Model of Chitter: https://github.com/rachjgriff/chitter-challenge/blob/master/UNADJUSTEDNONRAW_thumb_d77.jpg
+* Created Database structure for the Domain Model using Entity Relationship diagram: https://github.com/rachjgriff/chitter-challenge/blob/master/UNADJUSTEDNONRAW_thumb_d69.jpg
```

Comment:
I think you could have embedded this image and the one above rather than linking

Reply:
Why did you choose to have the images in the root directory rather than an image folder?